README.md

```diff
@@ -87,6 +87,8 @@ There are several important points to know about SendEmail:
     `cc` - email address or array of addresses
     `bcc` - email address or array of addresses
     `replyTo` - email address
+    `configurationSet` - SES configuration set name
+    `messageTags` - SES message tags: array of name/value objects, e.g. { name: xid, value: 1 }
```

Comment:
What do you think of using the `{ "xid": 1 }` structure instead of the more verbose `{ name: xid, value: 1 }`?

Reply:
I could go either way on this.  I did it this way because it's the first inspiration that came when looking at building the output structure.

You could have messageTags: {xid:1, uid:26, food:'pizza'} to generate three tags using the one object.  It does make multiple values for the same tag a pain, i.e. {xid:1, uid:26, food:'pizza',food:'cheeseburger'} would only have food=cheeseburger as the result.  I think that if you're going to go with the flat object, then that's the shortcoming - that you only have one value for one tag, or you have to add some logic to check for array values which would be more cumbersome.

If you're just talking about [ {xid:1}, {uid:26}, {food:'pizza'}, {food:'cheeseburger'} ] it's a matter of changing:
```
data['Tags.member.' + (i + 1) + '.Name'] = Object.keys(tagSpec)[0];
data['Tags.member.' + (i + 1) + '.Value'] = tagSpec[Object.keys(tagSpec)[0]];
```
or something equivalent and more sexxxxy.